app/assets/javascripts/comments.js

```diff
@@ -0,0 +1,16 @@
+// Place all the behaviors and hooks related to the matching controller here.
```

Comment:
Delete generated files if you don't use them. Also, don't leave comment-out blocks of code in master.